### PR TITLE
M2 pon wdl works in Firecloud; M2 wdl works in JES; updated oncotator docker in M2 template

### PR DESCRIPTION
**Pull request by davidbenjamin**
This PR resolves all the pre-release M2 stuff except for the launch script.

@LeeTL1220 @ruchim

### CODE REVIEW

File: scripts/mutect2_wdl/mutect2.wdl [hunk 1]
```diff
@@ -33,6 +33,7 @@ workflow Mutect2 {
   File tumor_bam_index
   File? normal_bam
   File? normal_bam_index
+    String vcf_name = basename(tumor_bam, ".bam") + ".vcf"
```
Comment (LeeTL1220) [inline scripts/mutect2_wdl/mutect2.wdl]: Can you rename to "output_vcf_name"?

Also, not sure if this is github.com website, but this line looks oddly indented.

File: scripts/mutect2_wdl/mutect2.wdl [hunk 2]
```diff
@@ -158,6 +160,28 @@ workflow Mutect2 {
 
 
   if (is_run_oncotator) {
+        call GetSampleName as GetTumorSample {
```
Comment (LeeTL1220) [inline scripts/mutect2_wdl/mutect2.wdl]: Why didn't you just output the sample names from the M2 task?  Wasn't that what was happening before?

File: scripts/mutect2_wdl/mutect2_template.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
   "Mutect2.is_run_orientation_bias_filter": "$__is_run_orientation_bias_filter__",
   "Mutect2.is_run_oncotator": "$__is_run_oncotator__",
   "Mutect2.gatk_docker": "broadinstitute/gatk:4.beta.5",
-  "Mutect2.oncotator_docker": "broadinstitute/oncotator:1.9.3.0",
+  "Mutect2.oncotator_docker": "broadinstitute/oncotator:1.9.6.1",
```
Comment (LeeTL1220) [inline scripts/mutect2_wdl/mutect2_template.json]: Thanks!

File: scripts/mutect2_wdl/mutect2_pon.wdl [hunk 4]
```diff
@@ -1,10 +1,11 @@
-import "mutect2_multi_sample.wdl" as m2_multi
+import "mutect2.wdl" as m2
```
Comment (LeeTL1220) [inline scripts/mutect2_wdl/mutect2_pon.wdl]: Can you put some preamble about the use of this WDL?

File: scripts/mutect2_wdl/mutect2.wdl (hunk 2 above)
Comment (LeeTL1220) [reply scripts/mutect2_wdl/mutect2.wdl]: No action required other than to answer the question.  Though I'd be grateful if we can avoid having extra localizing of bams.

**Comment by LeeTL1220** [comment]
@davidbenjamin Back to you

File: scripts/mutect2_wdl/mutect2_template.json (hunk 3 above)
Comment (samuelklee) [reply scripts/mutect2_wdl/mutect2_template.json]: The default Oncotator Docker in the CNV WDL is broadinstitute/oncotator:1.9.5.0-eval-gatk-protected, do we need to update?  I saw some chatter in the DSDE Comms Slack channel about the CNV pair workflow failing on FireCloud at the Oncotator step, not sure if this has anything to do with that.  @bshifaw

Comment (LeeTL1220) [reply scripts/mutect2_wdl/mutect2_template.json]: CNV docker is different than the M2 docker.  Each is correct in master.  

@bshifaw Make sure you have the latest WDL from `master`.  I made an update to fix an issue with the CNV docker.  I have confirmed that the fix works on ~12 samples.

File: scripts/mutect2_wdl/mutect2.wdl (hunk 1 above)
Comment (davidbenjamin) [reply scripts/mutect2_wdl/mutect2.wdl]: I indented due to my renegade convention to indent derived variables so people know not to put them in the json, but I can undo it.

File: scripts/mutect2_wdl/mutect2.wdl (hunk 2 above)
Comment (davidbenjamin) [reply scripts/mutect2_wdl/mutect2.wdl]: I threw the baby out with the bathwater.  I will restore the baby.

File: scripts/mutect2_wdl/mutect2_pon.wdl (hunk 4 above)
Comment (davidbenjamin) [reply scripts/mutect2_wdl/mutect2_pon.wdl]: done

File: scripts/mutect2_wdl/mutect2.wdl (hunk 1 above)
Comment (davidbenjamin) [reply scripts/mutect2_wdl/mutect2.wdl]: > Can you rename to "output_vcf_name"?

done.

**Pull request merged**